docs/conceptual/array.sum[^t]-function-[fsharp].md

```diff
@@ -14,9 +14,9 @@ ms.assetid: 532a4929-def0-4095-8c62-3bc7312038f2
 
 Returns the sum of the elements in the array.
 
-**Namespace/Module Path:** Microsoft.FSharp.Collections.Array
+**Namespace/Module Path**: Microsoft.FSharp.Collections.Array
```

Comment:
Is there a reason you changed this? The other docs (everywhere) keep the colon bold. I'd recommend not changing this.